Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsUITests.swift

```diff
@@ -247,46 +253,62 @@ final class FinancialConnectionsUITests: XCTestCase {
         // find + tap an institution; we add extra institutions in case
         // they don't get featured
         let institutionButton: XCUIElement?
-        let institutionTextInWebView: String?
+        let institutionName: String?
         let chaseInstitutionButton = app.webViews.buttons["Chase"]
         if chaseInstitutionButton.waitForExistence(timeout: 10) {
             institutionButton = chaseInstitutionButton
-            institutionTextInWebView = "Chase"
+            institutionName = "Chase"
         } else {
             let bankOfAmericaInstitutionButton = app.webViews.buttons["Bank of America"]
             if bankOfAmericaInstitutionButton.waitForExistence(timeout: 10) {
                 institutionButton = bankOfAmericaInstitutionButton
-                institutionTextInWebView = "Bank of America"
+                institutionName = "Bank of America"
             } else {
                 let wellsFargoInstitutionButton = app.webViews.buttons["Wells Fargo"]
                 if wellsFargoInstitutionButton.waitForExistence(timeout: 10) {
                     institutionButton = wellsFargoInstitutionButton
-                    institutionTextInWebView = "Wells Fargo"
+                    institutionName = "Wells Fargo"
```

Comment:
Are these values reused? should we define them as constants?

Reply:
@ccen-stripe 

I moved some of these to be less-copy-pasted:
- https://github.com/stripe/stripe-ios/pull/2854/commits/5f36b04e9ad2bdbd9a124a552dd370a2f3b0a59f

Also re-ran the tests and they passed.

I'm generally more loose on code for tests because as long as they do their job, it's OK. Tests often require code that's a bit funky anyway.